### ISSUE TITLE
FR: Python packages which use python setup.py for tests

### ISSUE THREAD

**Issue by jonasstein** [mm1ke/gentoo-scripts#27]
Tests should be run with esetup.py, but some ebuilds like
`sci-geosciences/tilecache`
try to run setup.py directly. This will lead to problems, if more than one python version is possible in the ebuild. In general there is no reason to use it and should be substituted by esetup.py

https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/

**Comment by mm1ke** [comment]
Ok, is this limited to tests only? setup.py seems to be used quite often..
At least ```qgrep -N " setup.py test"``` should be a good start.

**Comment by jonasstein** [comment]
Yes, the only the combination `" setup.py test"` is a hint that something could be wrong.
Of course it would lead to false positives in comments and so on. But I think we can find several broken py packages.
